documentation/en/curio-pdp.md

```diff
@@ -0,0 +1,105 @@
+---
+description: This page explains how to setup a Curio PDP SP
+---
+
+# Curio PDP
+
+This guide is focused on running a PDP-only SP.  Most of the same instructions will apply for running a curio node providing both PoRep and PDP storage though you will likely need to modify the configuration defaults specified here to keep the sealing pipeline running smoothly.
+
+## Setup curio 
+
+The first step to running a curio PDP SP is to install and setup curio.
+
+Some important callouts departing from the ordinary curio setup instructions
+* You MUST check out `feat/pdp` branch and build curio from source on this branch. PDP code is not yet on `main`
+* The `feat/pdp` branch has not been updated to go v1.24.x so running PDP SP requires building with go v1.23.x
+* Follow the guidelines under `Initiating a new Curio cluster` including the `guided-setup` command.  This means that you need to create a miner actor onchain (i.e. PoRep infrastructure). While unnecessary for PDP this is the quickest and easiest way to get PDP working on curio.
+
+
+With these points in mind proceed to the [curio setup instructions](setup.md#setup) to setup curio.
+
+## Curio configuration
+
+
+### Config 
+
+Curio PDP requires specific configuration options. Briefly you will need to modify your config to look like this: 
+
+```toml
+[HTTP]
+  DomainName = "your.domain.name.com"
+  Enable = true
+  IdleTimeout = 7200000000000
```

Comment:
All duration should be human friendly like "0h5m10s"

Reply:
Updated to human friendly here: https://github.com/filecoin-project/curio/pull/447/commits/919dc70bfc26bce3bc325392ca6f0ddaf9e226dc